.github/workflows/chart-release-pr.yml

```diff
@@ -54,4 +54,12 @@ jobs:
             --path="${{ env.CHART_PATH }}" \
             --config-file="${{ env.RELEASE_PLEASE_CONFIG_FILE }}" \
             --manifest-file="${{ env.RELEASE_PLEASE_MANIFEST_FILE }}" \
-            --debug
+            --debug | tee release-please.log
+      - name: Get release PR number
+        run: |
+          release_pr_number="$(grep "Successfully opened pull request:" release-please.log | grep -Po "\d+" || true)"
```

Comment:
In release-please.log , which I assume is the output of 
https://github.com/camunda/camunda-platform-helm/actions/runs/14094095761/job/39477607529
under Run Release Please, 

are there any other numbers that would match \d+  other than the PR number?

Reply:
Ah it's grepping the "Successfully opened pull request" first. my mistake

Reply:
Yeah, the pattern in the logs is `Successfully opened pull request: 1234.`